src/modules/complete_status.js

```diff
@@ -0,0 +1,22 @@
+import { storageData } from './store_locally.js';
+
+export const updateCompleteStatus = (todoList, completed, position) => {
+  const index = todoList.findIndex(
+    (item) => item.position === parseInt(position, 10),
+  );
+  if (index !== -1) {
+    todoList[index].completed = completed;
+  }
+  storageData(todoList);
+};
+
+export const deleteAllCompleted = (todoList) => {
+  const result = todoList.filter((res) => res.completed);
+  result.forEach((element) => {
+    const index = todoList.findIndex(
+      (item) => item.position === parseInt(element.position, 10),
+    );
+    todoList.splice(index, 1);
+  });
+  storageData(todoList);
+};
```

Comment:
The function filters the todoList to create a new array containing only completed items. Instead of using filter and then forEach, you can combine them into a single forEach loop using the filter condition directly.The function name could be more descriptive, such as removeAllCompletedTodos or deleteCompletedTodos.

Reply:
This has been resolved.